doc/rvv-intrinsic-spec.adoc

```diff
@@ -412,7 +412,7 @@ Non-mask (integer and floating-point) data types have SEW and LMUL encoded.
 
 Integer types have EEW and EMUL encoded into the type. The first row describes the EMUL and the first column describes the data type and element width of the scalar type.
 
-Types with subscript "^\*^" are available when `ELEN >= 64` (that is, unavailable under `Zve32*` and require at least `Zve64x`).
+Types with an asterisk (^\*^) are available when `ELEN >= 64` (that is, unavailable under `Zve32*` and require at least `Zve64x`).
```

Comment:
Let's use `({empty}*)` instead of  `(^\*^)` otherwise some of these asterisks get rendered correctly and others don't even in the same document (e.g. in my Ubuntu 22.04 this first one look OK but the later ones in line 444 and 459 do not).